addons/account/views/account_payment_term_views.xml

```diff
@@ -102,6 +102,45 @@
                             The last line's computation type should be "Balance" to ensure that the whole amount will be allocated.
                         </p>
                         <field name="line_ids"/>
+
+                        <!-- ============================== Early payment section ============================== -->
+                        <field name="early_payment_applicable" invisible="1"/>
+                        <div attrs="{'invisible':[('early_payment_applicable','=',False)]}">
+                            <field name="has_early_payment"/>
+                            <label for="has_early_payment"/>
+                            <div attrs="{'invisible':[('has_early_payment','=',False)]}">
+
+                                <div class="row mt-2">
+                                    <div class="col-2 mb-2">
+                                        <strong><label for="percentage_to_discount"/></strong>
+                                    </div>
+                                    <div class="col-6 mb-2">
+                                        <field name="percentage_to_discount"/>
+                                        <span class="oe_inline">%</span>
+                                    </div>
+                                </div>
+                                <div class="row">
+                                    <div class="col-2 mb-2">
+                                        <strong><label for="discount_computation"/></strong>
+                                    </div>
+                                    <div class="col-6 mb-2">
+                                        <field name="discount_computation"/>
+                                    </div>
+                                </div>
+                                <div class="row">
+                                    <div class="col-2 mb-2">
+                                        <strong><label for="discount_time_availability"/></strong>
+                                    </div>
+                                    <div class="col-6">
+                                        <field name="discount_days"/>
+                                        <field name="discount_time_availability" class="oe_inline"/>
+                                    </div>
+<!--                                    <div class="col-6 mb-2">-->
+<!--                                        <field name="discount_time_availability" class="oe_inline"/>-->
+<!--                                    </div>-->
```

Comment:
remove? :p

Reply:
that's currently in modification, a more final version will be up soon :)